zaza/openstack/charm_tests/cinder_backup_swift/setup.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2018 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Code for configuring cinder-backup-swift."""
+
+import zaza.model as zaza_model
+import time
+
+
+def basic_setup():
```

Comment:
There is no requirement for a ``basic_setup`` function, since it is empty here you could just remove it.

Reply:
removed